components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -3401,7 +3403,28 @@ subroutine ocn_compute_land_ice_flux_input_fields(layerThickness, normalVelocity
          ! the magnitude of the top drag is CD*u**2 = CD*(2*KE)
          topDragMag(iCell) = rho_sw * landIceFraction(iCell) &
                            * 2.0_RKIND * landIceTopDragCoeff *  kineticEnergyCell(minLevelCell(iCell),iCell)
+      end do
+#ifndef MPAS_OPENACC
+      !$omp end do
+      !$omp end parallel
+#endif
+
+      if ( trim(config_land_ice_flux_mode) == "data" ) then
+         ! top drag is set, and that's all we need in data (prescribed) mode
+         call mpas_timer_stop("land_ice_diagnostic_fields")
+         return
+      end if
+
+      ! From here on, were in "standalone" or "coupled" mode
 
+      ! compute friction velocity at cell centers
+#ifdef MPAS_OPENACC
+      !$acc parallel loop present(landIceFrictionVelocity, kineticEnergyCell, minLevelCell) &
+#else
```

Comment:
PGI (or nvhpc) compiler flagged the extra continuation char `&` as a build error when compiling for a [GPU run](https://my.cdash.org/viewTest.php?buildid=2351627):
```
NVFORTRAN-S-0292-Illegal continuation line (/gpfs/wolf/cli115/proj-shared/azamat/e3sm_scratch/SMS_Ld1.T62_oEC60to30v3.CMPASO-NYF.ascent_pgigpu.JNextGpuacc20230614_015956/bld/cmake-bld/core_ocean/shared/mpas_ocn_diagnostics.f90: 3071)
NVFORTRAN-S-0034-Syntax error at or near ACCDO (/gpfs/wolf/cli115/proj-shared/azamat/e3sm_scratch/SMS_Ld1.T62_oEC60to30v3.CMPASO-NYF.ascent_pgigpu.JNextGpuacc20230614_015956/bld/cmake-bld/core_ocean/shared/mpas_ocn_diagnostics.f90: 3070)
```
If I remove the ampersand, the case builds and runs OK.
@jonbob can you add this in the next PR you merge? Thanks.

Reply:
@amametjanov, I'm sorry about that.  Not sure how that crept in.

Reply:
@amametjanov -- I'll make a bug fix PR to handle this and make sure it gets in as soon as possible